src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets

```diff
@@ -0,0 +1,89 @@
+<!--
+***********************************************************************************************
+Microsoft.Net.Sdk.Razor.Versions.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+  <!--
+      Initialize properties and items inferred using the project version. This file is not imported in projects referencing
+      MVC \ Razor 2.2 or earlier since values specified here are provided via targets and props imported from NuGet packages
+      such as Microsoft.AspNetCore.Razor.Design.
+
+      The properties and items here are designed to be read by CPS so they should be just simple evaluation-time values
+      and should not require targets to initialize.
+  -->
+  <PropertyGroup>
+    <!--
+      MVC uses a ProvideApplicationPartFactoryAttribute on the generated assembly to load compiled views from assembly. Set this to false, to prevent generating this attribute.
+    -->
+    <GenerateProvideApplicationPartFactoryAttribute>true</GenerateProvideApplicationPartFactoryAttribute>
```

Comment:
Should this be in a .props? can the user's project override this

Reply:
Or have a condition

Reply:
Condition seems better. We want to set these up in 3.0 projects, a props file is too early for checking this.